src/script/backport-create-issue

```diff
@@ -239,16 +239,26 @@ def update_relations(r, issue, dry_run):
         if dry_run:
             logging.info(url(issue) + " add backport to " + release)
             continue
-        other = r.issue.create(project_id=issue['project']['id'],
-                               tracker_id=backport_tracker_id,
-                               subject=subject,
-                               priority_id=issue['priority']['id'],
-                               assigned_to_id=assigned_to_id,
-                               target_version=None,
-                               custom_fields=[{
-                                   "id": release_id,
-                                   "value": release,
-                               }])
+        create_args = {
+            "project_id": issue['project']['id'],
+            "tracker_id": backport_tracker_id,
+            "subject": subject,
+            "priority_id": issue['priority']['id'],
+            "assigned_to_id": assigned_to_id,
+            "target_version": None,
+            "custom_fields": [
+                {
+                    "id": release_id,
+                    "value": release,
+                }
+            ]
+        }
+        try:
+            other = r.issue.create(**create_args)
+        except ValidationError as e:
+            # try without an assignee
```

Comment:
Have you considered adding a WARNING log message for the retry?  No one is going to notice the POST line with 422 error code:
```
DEBUG:urllib3.connectionpool:https://tracker.ceph.com:443 "POST /projects/46/issues.json HTTP/1.1" 422 34
DEBUG:urllib3.connectionpool:https://tracker.ceph.com:443 "POST /projects/46/issues.json HTTP/1.1" 201 774
```
In this case, the issue with https://tracker.ceph.com/issues/66797 appears to be that Yite wasn't added to "Ceph Developers" group, and on top of that, only to the RADOS sub-project instead of to the Ceph project.  Having at least some record to catch such instances would be useful.

Reply:
> Have you considered adding a WARNING log message for the retry? No one is going to notice the POST line with 422 error code:
> 
> ```
> DEBUG:urllib3.connectionpool:https://tracker.ceph.com:443 "POST /projects/46/issues.json HTTP/1.1" 422 34
> DEBUG:urllib3.connectionpool:https://tracker.ceph.com:443 "POST /projects/46/issues.json HTTP/1.1" 201 774
> ```

I can but remember no one is reading this output anyway (unless something is broken). I'll do that in a followup.

> In this case, the issue with https://tracker.ceph.com/issues/66797 appears to be that Yite wasn't added to the "Ceph Developers" group, and on top of that was added only to the RADOS sub-project instead of to the Ceph project. Having at least some record to catch such instances would be useful.

I think generally the TL should reassign to someone who can be responsible for the backports in that case (since the backport is left unassigned). They can also use their judgement to add the original issue's author to the appropriate groups so they can be assigned.

Reply:
> I can but remember no one is reading this output anyway (unless something is broken).

Well, it's still to run it by hand and in general it's quite chatty.  Hiding a step such as a retry behind a DEBUG log message that differs only in a couple of characters is sneaky.

> I'll do that in a followup.

Fine with me.

Reply:
#58784